packages/tasks/src/model-libraries.ts

```diff
@@ -1022,6 +1022,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		docsUrl: "https://github.com/jasonppy/VoiceCraft",
 		snippets: snippets.voicecraft,
 	},
+	vui: {
+		prettyLabel: "VUI",
+		repoName: "VUI",
+		repoUrl: "https://github.com/vui-ai/vui",
+		countDownloads: `path_extension:"pt"`,
```

Comment:
Would be really good to nudge them to split their repo into several ones (better to track downloads on individual models). 

(highly recommended but not an obligation)